libraries/dynamic-ui-react/src/components/DAlertBox/DAlertBox.tsx

```diff
@@ -0,0 +1,60 @@
+import { useMemo } from 'react';
+import classNames from 'classnames';
+
+import type { PropsWithChildren } from 'react';
+
+import DIcon from '../DIcon';
+
+import type { BaseProps, AlertBoxTheme } from '../interface';
+
+type Props =
+  & BaseProps
+  & PropsWithChildren<{
```

Comment:
```suggestion
  & FamilyIconProps
  & PropsWithChildren<{
```

Remover abajo iconFamilyClass y prefix